controllers/openstackdataplane_controller.go

```diff
@@ -49,8 +46,23 @@ type OpenStackDataPlaneReconciler struct {
 	Log     logr.Logger
 }
 
+// DataPlaneAnnotations defines a struct that we can use to update annotations on Dataplane related
+// objects.
+type DataPlaneAnnotations struct {
+	// Name is just the string name used to reference the annotation
+	// this would typically serve as the key in the Annotations map
+	Name string
+
+	// Annotations represents the map of annotations that should be
+	// applied to the object.
+	Annotations map[string]string
+}
+
+// edpmDeployAnnotationName - This annotation name is used to trigger Ansible deployments against dataplane nodes.
+const edpmDeployAnnotationName string = "edpm.openstack.org/deploy"
+
```

Comment:
here as well, let's drop the annotation changes. I don't think an annotation is the way to go.